src/Nethermind/Nethermind.Runner/Ethereum/Steps/InitializeBlockchainAuRa.cs

```diff
@@ -170,8 +173,8 @@ private IAuRaValidator CreateAuRaValidator(IBlockProcessor processor)
                             GetReadOnlyTransactionProcessorSource()))
                         .ToArray<IBlockGasLimitContract>(),
                     _api.GasLimitCalculatorCache,
-                    _api.Config<IAuraConfig>().Minimum2MlnGasPerBlockWhenUsingBlockGasLimitContract,
-                    new TargetAdjustedGasLimitCalculator(_api.SpecProvider, _api.Config<IMiningConfig>()), 
+                    ((INethermindApi)_api).Config<IAuraConfig>().Minimum2MlnGasPerBlockWhenUsingBlockGasLimitContract,
```

Comment:
cast not needed?

Reply:
it is silly - > Config<> is defined on the interface and not on the base type